docs/source/http_services.rst

```diff
@@ -52,7 +52,11 @@ Pages and JSON endpoints served by the scheduler
 Scheduler API
 -------------
 
-Scheduler methods exposed by the API with an example of the request body they take
+Scheduler methods exposed by the API with an example of the request body they take.
+
+.. note::
+   To prevent unauthorized access, the scheduler API is disabled by default if `tls`_ is enabled.
+   See the ``distributed.http.insecure-routes`` :doc:`config <configuration>` setting.
```

Comment:
The API is disabled if `tls` is **disabled**. This sentence is the other way round, isn't it?